src/App.js

```diff
@@ -73,12 +72,11 @@ const App = () => {
 
   const classes = useStyles();
 
-  const onSuccess = ({ img, secure }) => {
-    setImage(img);
-    const fcmResponse = { img, secure };
```

Comment:
The `onSuccess` callback already returns the `payload` that contains `secure` in case the mode is enabled and it is used [here](https://github.com/getyoti/web-fcm-demo/pull/30/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R79) to build the final request so the `fcmResponse` variable is not needed anymore.

Reply:
Nothing actually changed here, the object was unwrapped before and then wrapped, and now it is never unwrapped.

Previous `fcmResponse` === new `payload`

Reply:
thanks for the clarification